index.yaml

```diff
@@ -1062,28 +1062,6 @@ entries:
     urls:
     - assets/fleet/fleet-107.0.0+up0.13.0-alpha.6.tgz
     version: 107.0.0+up0.13.0-alpha.6
-  - annotations:
```

Comment:
I'm assuming this was automatically removed. Do we need this included?

Reply:
we dont update index.yaml manually.. the update happens as part of `make chart` process, so those changes are left as is..